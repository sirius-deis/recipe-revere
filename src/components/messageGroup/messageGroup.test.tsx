import { render, screen } from "@testing-library/react";
import MessageGroup from "./messageGroup";


describe("MessageGroup component", () => {
  it("should match snapshot", () => {
    const { container } = render(<MessageGroup messages={[]} sender="sender" />);
    expect(container).toMatchSnapshot();
  });
  it("should render messages", () => {
    const messages = [
      { _id: "1", message: "Hello", sender: "sender1", timestamp: "1626954246277", isRead: false, isMine: true },
      { _id: "2", message: "World", sender: "sender2", timestamp: "1626954260956", isRead: true, isMine: false },
    ];
    render(<MessageGroup messages={messages} sender="sender" />);
    expect(screen.getByText("Hello")).toBeInTheDocument();
    expect(screen.getByText("World")).toBeInTheDocument();
  });
});