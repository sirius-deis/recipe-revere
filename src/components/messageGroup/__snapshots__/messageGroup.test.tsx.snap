// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageGroup component should match snapshot 1`] = `
<div>
  <div
    class="message-group"
  />
</div>
`;
